assignment2.txt

```diff
@@ -0,0 +1,11 @@
+Bettina Benitez
+
+1. You can combine two or more operators. However, you must include && or ||.
+2. Logical errors happen when the output of the code is unexpected while Syntax errors are grammatical errors in the code.
+3. The conditional statement is true
```

Comment:
Correct, but remember that boolean values don't exist in C, so the answer is technically 1!